examples/dreambooth/README_flux.md

```diff
@@ -191,5 +208,16 @@ accelerate launch train_dreambooth_lora_flux.py \
   --push_to_hub
 ```
 
+## Memory Optimizations
```

Comment:
This should, IMO, include:

* Decrease image resolution
* 8bit optimizer
* Gradient checkpointing and accumulation 
* Latent caching 

WDYT?

Reply:
we use `--resolution=512` by default and also in the readme examples, so I wonder if it adds much to that section?

Reply:
Perhaps making the point more explicit could help the users think about this aspect more?